0.data_analysis_and_processing/mic_analysis/nbconverted/well_sirna_mic_analysis.py

```diff
@@ -240,13 +243,13 @@
 
     plt.gcf().set_size_inches(18, 10)
 
-    plt.xlabel("MIC", fontsize=13)
-    plt.ylabel("Density", fontsize=13)
+    plt.xlabel("MIC Score", fontsize=13)
+    plt.ylabel("Number of Wells", fontsize=13)
```

Comment:
Nice update! Number of wells makes much more sense!

Reply:
Agreed